src/cluster.h

```diff
@@ -173,257 +53,83 @@ typedef struct {
     redisDb *db;                /* A link back to the db this dict belongs to */
 } clusterDictMetadata;
 
+/*-----------------------------------------------------------------------------
+ * Cluster public types
+ *----------------------------------------------------------------------------*/
+typedef intptr_t clusterNodeHandle;
```

Comment:
Minor preference towards `typedef struct clusterNodeHandle *clusterNodeHandle;`, so that it's clear the handle is a pointer. I find a lot of people get confused at how sds is a really pointer.